packages/mailer/handlers/studio/src/index.ts

```diff
@@ -0,0 +1,42 @@
+import { AbstractMailHandler } from '@redwoodjs/mailer-core'
+import type {
+  MailRenderedContent,
+  MailResult,
+  MailSendOptionsComplete,
+} from '@redwoodjs/mailer-core'
+import { NodemailerMailHandler } from '@redwoodjs/mailer-handler-nodemailer'
+import type { HandlerOptions } from '@redwoodjs/mailer-handler-nodemailer'
+
+export class StudioMailHandler extends AbstractMailHandler {
+  private nodemailerHandler: NodemailerMailHandler
+
+  constructor() {
+    super()
+
+    this.nodemailerHandler = new NodemailerMailHandler({
+      transport: {
+        host: 'localhost',
+        port: 4319,
```

Comment:
I forget, is the Studio port configurable? If so, if I change will this match up?

Reply:
Right now none of the studio ports are configurable so this should be fine as it is until I allow this to be configured in the future